torch/csrc/lazy/ts_backend/ir_builder.cpp

```diff
@@ -0,0 +1,16 @@
+#include <torch/csrc/lazy/backend/backend_interface.h>
+#include <torch/csrc/lazy/ts_backend/ir_builder.h>
+
+// This file contains the TorchScript IrBuilder
+
+namespace torch {
+namespace lazy {
+
+// Get IrBuilder from backend. Use TorchScriptIrBuilder by default
+const IrBuilder* getIrBuilder() {
```

Comment:
maybe move this file to csrc/lazy/backend,
and just have it do
TORCH_CHECK(hasBackend()) then always use getBackend()->GetIrBuilder()
?